.github/workflows/docker_cell-type-ewings.yaml

```diff
@@ -45,15 +47,23 @@ jobs:
         uses: docker/setup-buildx-action@v3
 
       - name: Build image
-        uses: docker/build-push-action@v5
+        uses: docker/build-push-action@v6
         with:
           context: "{{defaultContext}}:analyses/simulate-sce"
           push: false
           cache-from: type=gha
           cache-to: type=gha,mode=max
 
-  build-push:
-    name: Build and Push Docker Image
+  build:
+    name: Build Docker Image
+    if: github.repository_owner == 'AlexsLemonade' && (github.event_name == 'push' || inputs.push-ecr)
+    uses: ./.github/workflows/build-push-docker-module.yml
+    with:
+      module: cell-type-ewings
+      push-ecr: false
+
+  push:
+    name: Push Docker Image
```

Comment:
I think you might have to make the change for this in the  `build-push-docker-module.yml` file itself

Reply:
Okay I made that change in https://github.com/AlexsLemonade/OpenScPCA-analysis/pull/690/commits/5c1d29b3e1022ea584f4e027d1c550c2bec4a1ba. Do we want to merge this in so that it will actually test the changes...?

Reply:
You will need to revert the changes in this section (go back to `build-push`)